pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -538,6 +568,9 @@ func addDestinationServiceAttributes(attrs attributes, push *model.PushContext,
 
 	svc := push.ServiceByHostname[destinationHostname]
 	if svc == nil {
+		if destinationHostname == util.PassthroughCluster || destinationHostname == util.BlackHoleCluster {
+			attrs["destination.service.name"] = attrStringValue(string(destinationHostname))
```

Comment:
You probably can leave this out. `destination.service.host` is already populated, and it's sufficient for telemetry.

Reply:
It's not sufficient as for many metrics only `destination.service.name` is used as `destination.service` is deprecated.

Issue is because `Passthrough` or `Blackhole` don't follow the host naming convention the `destination.service.name` attribute doesn't get populated in all cases hence doing it here.

Reply:
OK, good to know. This is fine then.